test/e2e/scheduling/OWNERS

```diff
@@ -1,4 +1,5 @@
 approvers:
 - sig-scheduling-maintainers
+- k82cn
```

Comment:
As this is the only place `sig-scheduling-maintainers` is used, perhaps it would be cleaner to just add your name to `sig-scheduling-maintainers` in `OWNERS_ALIASES`?